docs/docs/Getting-Started/getting-started-installation.md

```diff
@@ -1,97 +1,115 @@
 ---
-title: 📦 Installation
+title: Install Langflow
 sidebar_position: 1
 slug: /getting-started-installation
 ---
 
+You can deploy Langflow either locally or as a hosted service with [**Datastax Langflow**](#datastax-langflow).
 
+## Install Langflow Locally
 
-## **DataStax Langflow** {#a3b4bd8dec5a49ebbfca4828492133e9}
+Install Langflow locally with [pip](https://pypi.org/project/pip/) or [pipx](https://pipx.pypa.io/stable/installation/).
 
+### Prerequisites
 
----
-
-
-The easiest way to get started with Langflow is through the DataStax Cloud Service!
+* [Python 3.10 to 3.12](https://www.python.org/downloads/release/python-3100/) installed
+* [pip](https://pypi.org/project/pip/) or [pipx](https://pipx.pypa.io/stable/installation/) installed
+* Before installing Langflow, we recommend creating a virtual environment to isolate your Python dependencies with [venv](https://docs.python.org/3/library/venv.html) or [conda](https://anaconda.org/anaconda/conda)
 
+### Install Langflow
 
+Install Langflow with pip:
 
-**DataStax Langflow** is a hosted version of Langflow integrated with [Astra DB](https://www.datastax.com/products/datastax-astra). Be up and running in minutes with no installation or setup required. [Sign up for free](https://astra.datastax.com/signup?type=langflow).
-
-
-![](./602374500.png)
-
-
-## **Install Langflow Locally** {#ef364ee864c545649d248113ad7d3038}
-
+```bash
+python -m pip install langflow
+```
 
----
+Install Langflow with pipx using the Python 3.10 executable:
 
+```bash
+pipx install langflow --python python3.10
+```
 
-:::caution
+## Run Langflow
 
-Langflow **requires** Python version 3.10 or greater and [pip](https://pypi.org/project/pip/) or [pipx](https://pipx.pypa.io/stable/installation/) to be installed on your system.
+1. To run Langflow, enter the following command.
 
-:::
+```bash
+python -m langflow run
+```
 
+2. Confirm that a local Langflow instance starts by visiting `http://127.0.0.1:7860` in a Chromium-based browser.
 
+3. Continue on to the [Quickstart](/getting-started-quickstart).
```

Comment:
Avoid using numbered steps for things that aren't required. Suggest making this a paragraph below step 2 that says something like "Now that Langflow is running, follow the [quickstart](/getting-started-quickstart) to create your first flow."